bin/check_style

```diff
@@ -32,10 +32,17 @@ failure=
 results=
 count=0
 for file in forch/*.py testing/python_lib/*.py; do
-    echo Checking file $file... | tee -a $RESULTS
-    pylint --rcfile=$PYLINTRC $file >> $RESULTS && count=$((count+1)) || true
+    echo Checking file $file... | tee -a $RESULTS.$count
+    pylint --rcfile=$PYLINTRC $file >> $RESULTS.$count &
```

Comment:
this can be just > instead of >>
in case those files were removed from a previous unfinished run.

Reply:
Updated. PTAL. Tests are failing but I think that's a not-this-PR problem (normal build is failing too from what I can tell).